software/flyimg.yml

```
@@ -0,0 +1,12 @@
+name: "Flyimg"
+website_url: "https://flyimg.io"
+source_code_url: "https://github.com/flyimg/flyimg"
+description: "Dockerized application to resize and crop images on the fly. Get optimised images with MozJPEG, WebP or PNG using ImageMagick, with an efficient caching system."
```

As Docker is the only platform, this cleans the description.

```suggestion
description: "Resize and crop images on the fly. Get optimised images with MozJPEG, WebP or PNG using ImageMagick, with an efficient caching system."
```